spec/functions/README.md

```diff
@@ -16,13 +15,22 @@
    1. [`:date`](datetime.md#the-date-function)
    1. [`:time`](datetime.md#the-time-function)
 
-## Introduction
-
 This section defines the **_<dfn>default functions</dfn>_**
 which are REQUIRED for conformance with this specification,
 along with _default functions_ that SHOULD be implemented to support
 additional functionality.
 
+> [!IMPORTANT]
+> _Functions_ that are not marked **Draft** are **Stable** and subject to
+> the provisions of the [stability policy](../intro.md#stability-policy).
+>
+> _Functions_ and _options_ marked as **Draft** are not stable.
+> Their name, _operands_, and _options_/_values_, and other requirements
+> might change or be removed before being declared **Stable** in a future release.
+>
+> Some **Draft** _functions_ or _options_ are also marked as **Proposed**.
```

Comment:
you use 'or' here, but 'and' 5 lines above. Do you mean to contrast them?

Reply:
The difference between 'Draft' and 'Proposed' is unclear.

Reply:
Do we really need two different statuses for non-Stable things at this point? Especially as either Draft or Proposed status enables anything about them to change.

Reply:
@eemeli Actually, I think we might... not so much in this release as in the future.

The WG has formally approved a number of these, including their text, options, values, and such. These are effectively in (extended) Technical Preview and presumably lowercase-s stable. 

Items that the WG has not accepted yet need to be differentiated. The one example in this release is the option `timeZone`.

That said, I could live with everything being Draft and can make that change.

Reply:
@macchiati I meant to use `or` since there are options inside stable functions that are draft.

Reply:
I'd prefer to go with a single Draft definition for now, and then revisit when we know better what the '48 spec will look like.

Reply:
Fair enough. I'll make that change now.